charts/immich/values.yaml

```diff
@@ -1,268 +1,135 @@
+typesense:
+  enabled: false
+  env:
+    TYPESENSE_DATA_DIR: /tsdata
+    TYPESENSE_API_KEY: typesense
+  persistence:
+    tsdata:
+      enabled: false
+      accessMode: ReadWriteOnce
+      size: 1Gi
+      # storageClass: storage-class
+  image:
+    repository: docker.io/typesense/typesense
+    tag: 0.24.0
+    pullPolicy: IfNotPresent
+
+# Immich components
 
 server:
   enabled: true
   env:
     NODE_ENV: "production"
-  service:
-    main:
-      enabled: true
-      primary: true
-      type: ClusterIP
-      ports:
-        http:
-          enabled: true
-          primary: true
-          port: 3001
-          protocol: HTTP
   image:
-    # -- image repository
-    repository: altran1502/immich-server
-    # -- image tag
-    tag: v1.49.0
-    # -- image pull policy
+    repository: ghcr.io/immich-app/immich-server
     pullPolicy: IfNotPresent
-  command: "/bin/sh"
-  args:
-    - "./start-server.sh"
 
-microservice:
+microservices:
   enabled: true
   env:
     NODE_ENV: "production"
-  service:
-    main:
-      enabled: false
-      primary: true
-      type: ClusterIP
-      ports:
-        http:
-          enabled: true
-          primary: true
-          port: 3001
-          protocol: HTTP
+    REVERSE_GEOCODING_DUMP_DIRECTORY: /geodata-cache
+  persistence:
+    geodata-cache:
+      enabled: true
+      size: 1Gi
+      # Optional: Set this to pvc to avoid downloading the geodata every start.
+      type: emptyDir
+      accessMode: ReadWriteMany
+      # storageClass: your-class
   image:
-    # -- image repository
-    repository: altran1502/immich-server
-    # -- image tag
-    tag: v1.49.0
-    # -- image pull policy
+    repository: ghcr.io/immich-app/immich-server
     pullPolicy: IfNotPresent
-  command: "/bin/sh"
-  args:
-    - "./start-microservices.sh"
 
-machine_learning:
+machine-learning:
   enabled: true
   image:
-    # -- image repository
-    repository: altran1502/immich-machine-learning
-    # -- image tag
-    tag: v1.49.0
-    # -- image pull policy
+    repository: ghcr.io/immich-app/immich-machine-learning
     pullPolicy: IfNotPresent
-
-  service:
-    main:
-      enabled: true
-      primary: true
-      type: ClusterIP
-      ports:
-        http:
-          enabled: true
-          primary: true
-          port: 3003
-          protocol: HTTP
   env:
-    NODE_ENV: "production"
-    TRANSFORMERS_CACHE: /usr/src/app/.transformers_cache
+    TRANSFORMERS_CACHE: /cache
+  persistence:
+    cache:
+      enabled: true
+      size: 10Gi
+      # Optional: Set this to pvc to avoid downloading the ML models every start.
+      type: emptyDir
+      accessMode: ReadWriteMany
+      # storageClass: your-class
 
 web:
   enabled: true
   image:
-    # -- image repository
-    repository: altran1502/immich-web
-    # -- image tag
-    tag: v1.49.0
-    # -- image pull policy
+    repository: ghcr.io/immich-app/immich-web
     pullPolicy: IfNotPresent
   env:
     NODE_ENV: "production"
```

Comment:
On the next release NODE_ENV shouldn't be needed here

Reply:
Correct. I'll wait for that release to drop before removing it though (probably today).